eng/scripts/Update-Snippets.ps1

```diff
@@ -19,8 +18,20 @@ if ($ServiceDirectory -and ($ServiceDirectory -ne "*")) {
 
 if (-not (Test-Path env:TF_BUILD)) { $StrictMode = $true }
 
-if($StrictMode) {
-    Resolve-Path "$root" | %{ dotnet run -p $generatorProject -b "$_" -sm -c Release }
-} else {
-    Resolve-Path "$root" | %{ dotnet run -p $generatorProject -b "$_" -c Release }
+nuget install SnippetGenerator -Source azure-sdk-tools@DevOps -PreRelease -OutputDirectory .
+$generatorAssembly = Resolve-Path -Path ".\SnippetGenerator*\tools\**\SnippetGenerator.dll"
```

Comment:
Why do we need to resolve the path? After you install a global tool you should be able to use it from the path.

Reply:
Am actually installingit directly to the current directory then cleaning up after.

Reply:
I understand you are installing it locally but you should still be able to run it without needing to glob and find the path like you are doing here.

Reply:
Is there any reason the tool doesn't have a exe in the install location? Or any reason you cannot do `dotnet run .\SnippetGenerator`

Reply:
Looks like you didn't package this as a tools package see https://github.com/Azure/azure-sdk-tools/blob/main/tools/pipeline-generator/Azure.Sdk.Tools.PipelineGenerator/Azure.Sdk.Tools.PipelineGenerator.csproj#L7 which is what you need to get a standard dotnet tools package.